packages/host/app/resources/file.ts

```diff
@@ -226,10 +226,10 @@ class _FileResource extends Resource<Args> {
         let clientRequestId = event.clientRequestId;
         let reloadFile = false;
 
-        if (!clientRequestId) {
+        if (!clientRequestId || clientRequestId.startsWith('instance:')) {
```

Comment:
`instance:` prefixed client request id's originate from the preview editor. the file resource should always refresh on invalidations originating from instance updates.